mne/channels/channels.py

```diff
@@ -755,15 +755,17 @@ def drop_channels(self, ch_names):
         -----
         .. versionadded:: 0.9.0
         """
-        if isinstance(ch_names, list):
-            if not all([isinstance(ch, str) for ch in ch_names]):
-                raise ValueError("'ch_names' must be a list of strings, got "
-                                 "{}.".format([type(ch) for ch in ch_names]))
-        elif isinstance(ch_names, str):
+        if isinstance(ch_names, str):
             ch_names = [ch_names]
+
+        try:
```

Comment:
use _validate_types?

Reply:
But then I'd have to specify specific types, whereas here I can pass anything that quacks like a list.

Reply:
Hm. Maybe we should add „iterable“ to _valid_types. Just if something has __next__ or something like that. 

Not sure.

Reply:
Maybe but let's do it in a follow-up when someone is bored